src/__tests__/createClient.js

```diff
@@ -0,0 +1,211 @@
+import createClient from '../createClient';
+import { SUCCESS_URL, FAILING_URL } from '../__mocks__/openWebSocket';
+
+
+jest.mock('../openWebSocket');
```

Comment:
I can see what this must do - we're going to end up with our tests running with your mocked WebSocket implementation -  but I'm not exactly sure how it achieves that given the import ../createClient above itself imports the real ../openWebSocket. Perhaps you could explain it to me in Slack..